src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bars">
+      <form
+        action="local"
+        method="post"
+        data-qa="big"
+      >
+        <input
+          class="bar big__bar"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        action="local"
```

Comment:
Similar to the previous form, the 'action' attribute is set to 'local'. Ensure that you provide a valid URL or path for form submission, or remove the 'action' attribute if the form is not intended to be submitted.